tests/ledger_sequence_increment.sql

```diff
@@ -0,0 +1,33 @@
+{{ config(severity="warn"
```

Comment:
Where is `tests/ledger_sequence_increment.sql` used?

Reply:
Syd asked if we could add a test to ensure that the sequence is always incrementing, so we don't have a case where max(sequence) of batch_id > max(sequence) of batch_id + 1 (next batch id). Since the packages we're using didn't have a test to check this case, I configured as a dbt singular test. Let me know if this makes sense to you. Thanks!